platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/AddressBookInitializer.java

```diff
@@ -97,18 +101,20 @@ public class AddressBookInitializer {
      * @param softwareUpgrade   Indicate that the software version has upgraded.
      * @param signedState       The signed state loaded from disk.  May be null.
      * @param configAddressBook The address book derived from config.txt. Must not be null.
-     * @param addressBookConfig The configuration settings for AddressBooks.
+     * @param platformContext   The context for the platform. Must not be null.
```

Comment:
@NonNull and @Nullable annotation are part of the JavaDoc. Therefore I do not get why you add the "Must not be null" text to all params.

Reply:
Copilot auto-complete based off of existing comments.   I'll remove them from all comments.